go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/miekg/dns
 go 1.12
```

Comment:
You should bump this to the version that introduced crypto/ed25519.

Reply:
My bad, sorry about that. Thanks